com.unity.multiplayer.mlapi/Tests/Runtime/NetworkBehaviourUpdaterTests.cs

```diff
@@ -155,14 +151,6 @@ void AddNetworkBehaviour(Type type, GameObject prefab)
                 }
                 Assert.That(nbVarsCheckedClientSide, Is.EqualTo(m_ClientNetworkManagers.Length > 0 ? serverNetVarCount : 0));
             }
-
-            if (serverNetVarCount > 0 && nbClients > 0) // check for nb clients, this is a useless test if not clients, netvars remain dirty always
```

Comment:
@SamuelBellomo I'm not sure whether I'm breaking this test here and removing too much. Essentially we have a guarantee now that NetworkBehaviourUpdate will only be called once per tick by the tick system so this check will be no longer necessary I think?

Reply:
That guarantee should still be tested somewhere, right? With my quick overview of your tests I didn't see it, but I might have skipped over it too quickly.

Reply:
There is a test which ensures that the right amount of tick events are invoked as our network tick increases. I think that should be good enough to give us a guarantee that this works? We could still keep this test here to ensure that in addition this individual part also only gets called once but it means keeping additional code in the NetworkBehaviourUpdater which is really only used for tests.